ihatemoney/tests/main_test.py

```diff
@@ -385,9 +385,9 @@ def test_only_one_instance(self):
         assert one == two
 
     def test_get_currencies(self):
-        assert set(self.converter.get_currencies()) == set(
-            ["USD", "EUR", "CAD", "PLN", CurrencyConverter.no_currency]
-        )
+        currencies = self.converter.get_currencies()
+        for currency in ["USD", "EUR", "CAD", "PLN", CurrencyConverter.no_currency]:
```

Comment:
I'm a bit confused with `CurrencyConverter.no_currency` here. Should it be `XXX`?

Reply:
Well yes, it's the same thing.

Reply:
[Heh, yes it is](https://github.com/spiral-project/ihatemoney/pull/1240/files#diff-9534886c8df967ba5f34f0c976eb6790ec62b2e39103822885bff66ad95a2d4cR19)